CONTRIBUTING.md

```diff
@@ -246,6 +246,17 @@ All donated code requires a license compatible with the Apache Software License
 2.0, and donated code will require a change of copyright to reflect the
 OpenTelemetry Authors.
 
+### Donation process
+
+Broadly, these are the steps the OpenTelemetry Governance and Technical
+Committees follow to handle a prospective donation.
+
+1. Per the above, the donating organization creates a GitHub issue (with the appropriate template) in the `community` repository.
+2. The Technical Committee (TC) will do diligence, write a report, and attach that report to the GitHub issue
+3. The Governance Committee (GC) will consider the report and make a final decision about the donation
```

Comment:
What is the GC deciding? To accept or deny the donation, or to simply continue the process? At this point may the GC also ask the TC for additional clarification on their report or for additional information or steps? Or should that be done during the TC's initial technical reporting period?

Reply:
```suggestion
3. The Governance Committee (GC) will consider the report and make a final decision about the donation. GC must ensure that the donation is aligned with the project vision and roadmap and has a balanced set of interested contributors and maintainers. GC also responsible to drive awareness in the community about the contribution and making sure all interested parties had a chance to object and/or contribute in the scope of a donation.
```